pegasus/sites.v3/code.org/views/dance_teacher_resources.haml

```diff
@@ -7,14 +7,14 @@
       description: I18n.t(:hoc2018_dance_teacher_resources_tile1_description),
       image: "/images/dance-hoc/dance_party_dance.gif",
       button_text: I18n.t(:hoc2018_dance_teacher_resources_tile1_button),
-      link: CDO.studio_url('/s/dance')
+      link: "https://curriculum.code.org/hoc/plugged/8/"
```

Comment:
Nit: I tend to prefer to leave off the extra `/` at the end since it's unnecessary.